vllm/entrypoints/openai/api_server.py

```diff
@@ -63,6 +67,9 @@ async def check_model(request) -> Optional[JSONResponse]:
 
 
 async def get_gen_prompt(request) -> str:
+    assert _fastchat_available, (
```

Comment:
nit: As this is user-facing error, I feel raising error is a bit more appropriate.

> assert is used to ensure internal correctness, not to enforce correct usage nor to indicate that some unexpected event occurred. If an exception is desired in the latter cases, use a raise statement.